flows/vision-zero/cris_import/cris_import.py

```diff
@@ -311,17 +365,22 @@ def remove_trailing_carriage_returns(data_loaded_token):
         user=DB_USER, 
         password=DB_PASS, 
         dbname=DB_NAME, 
-        sslmode="require", 
+        sslmode=DB_SSL_REQUIREMENT, 
         sslrootcert="/root/rds-combined-ca-bundle.pem"
         )
 
-    columns = util.get_input_tables_and_columns(pg, DB_IMPORT_SCHEMA)
+    columns = util.get_input_tables_and_columns(pg, map_state["import_schema"])
```

Comment:
This variable is going to get entirely written out of the app, and you'll see this change a bunch below.